Compiling error in m_joinpartspam module

When trying to compile this module on latest git of insp3 branch i got the following:

```bash
BUILD:		modules/m_joinpartspam.cpp
/home/inspircd/repo_inspircd/src/modules/m_joinpartspam.cpp:289:7: error: ‘void JoinPartSpam::SerializeParam(Channel*, joinpartspamsettings*, std::__cxx11::string&)’ marked ‘override’, but does not override
  void SerializeParam(Channel* chan, joinpartspamsettings* jpss, std::string& out) CXX11_OVERRIDE
       ^~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
real.mk:672: recipe for target 'modules/m_joinpartspam.so' failed
make[1]: *** [modules/m_joinpartspam.so] Error 1
GNUmakefile:158: recipe for target 'target' failed
make: *** [target] Error 2
```

--- a/3.0/m_joinpartspam.cpp
+++ b/3.0/m_joinpartspam.cpp
@@ -286,7 +286,7 @@ class JoinPartSpam : public ParamMode<JoinPartSpam, SimpleExtItem<joinpartspamse
 		return MODEACTION_ALLOW;
 	}
 
-	void SerializeParam(Channel* chan, joinpartspamsettings* jpss, std::string& out) CXX11_OVERRIDE
+	void SerializeParam(Channel* chan, joinpartspamsettings* jpss, std::string& out)
 	{
 		out.append(ConvToStr(jpss->cycles)).push_back(':');
 		out.append(ConvToStr(jpss->secs)).push_back(':');

